packages/package-version-parser/package-version-parser.js

```diff
@@ -5,10 +5,8 @@
 // true).
 var inTool = typeof Package === 'undefined';
 
-
-var semver = inTool ?
-  require ('../../dev_bundle/lib/node_modules/semver') : SemVer410;
-var __ = inTool ? require('../../dev_bundle/lib/node_modules/underscore') : _;
+// Provided by dev_bundle/server-lib/node_modules/semver.
+var semver = require('semver');
```

Comment:
@benjamn - Looking at the failing tests, could we just leverage the `module.parent` here, if available, to make sure `semver` is pulled in when coming from `tools/packaging/package-version-parser.js`? So change 

```js
var semver = require('semver');
```

to something like:

```js
var moduleParent = module.parent;
var semver =
  moduleParent && moduleParent.require
    ? moduleParent.require('semver') : require('semver');
```